src/components/buttonsIndicators/buttonSecondary/index.tsx

```diff
@@ -15,8 +15,8 @@ export interface Props {
   icon?: {image: React.ReactNode, position: 'left' | 'right'}
 }
 
-export type Color = 'brand' | 'action' | 'subtle'
-export type Size = 'large' | 'medium' | 'small'
+export type Color = 'brand' | 'action' | 'subtle' | 'donate'
```

Comment:
New color should never be added directly to common component if it's not added to the design language first. If we need a new color, please ping in design channel and we can talk about it there. For now please use the closes one that you can find, the same way as you did for the welcome page